Teroy.js

```diff
@@ -73,38 +73,67 @@
       this.rendered = true;
     }
 
+    /**
+     * 
+     * @param {NamedNodeMap} newAttrs 
+     * @param {NamedNodeMap} oldAttrs 
+     * @param {Node} node
+     */
+    diffAttributes(newAttrs, oldAttrs, node) {
```

Comment:
In the finished code these comments should be removed. However, maybe we could create a Teroy.max.js file? Here we could in more depth comment and explain the code, and maybe we could create a flowchart to explain how/why everything works so automatically with the Proxy object?

Reply:
I like the idea of a `max` file, I can remove the comments and maybe we'll do another PR with the `max` version?